Storage.md

```diff
@@ -0,0 +1,261 @@
+This guide will describe how FullScreenShenanigans projects store and keep track of game information.
+
+### Table of Contents
+1. [ItemsHoldr](#itemsholdr)
+    1. [Local Storage](#local-storage)
+    2. [Getters and Setters](#getters-and-setters)
+    3. [Updating](#updating)
+    4. [Clearing and Defaults](#clearing-and-defaults)
+    5. [Elements](#elements)
+2. [StateHoldr](#stateholdr)
+
+# ItemsHoldr
+
+ItemsHoldr is a versatile container to store and manipulate values in localStorage, and optionally keep an updated HTML container showing these values.
+
+## Local Storage
+
+Local Storage is a browser property where information is saved past session activity.
+ItemsHoldr keeps track of items and their values while the game is running and periodically saves them to `localStorage`.
+
+Because not just these FSS projects might be using the browser's localStorage, ItemsHoldr has a prefix property to differentiate which information belongs to which source.
+
+```typescript
+let ItemsHolder: ItemsHoldr = new ItemsHoldr({
+    prefix: "FullScreenShenanigans"
+});
+```
+
+## Getters and Setters
+
+ItemsHoldr has a number of functions that return information about the collection of items or a specific item.
+
+```typescript
+// simple gets
+ItemsHolder.getKeys();
+ItemsHolder.getValues();
+ItemsHolder.exportItems();
+ItemsHolder.getAutoSave();
+ItemsHolder.getLocalStorage();
+ItemsHolder.getPrefix();
+ItemsHolder.getContainer();
+ItemsHolder.getDisplayChanges();
+
+// returns the indexed key
+ItemsHolder.key(1);
+// returns the value under the key
+ItemsHolder.getItem("color");
+// returns whether the key is in the collection
+ItemsHolder.hasKey("color");
+```
+
+To add an item to the collection, use `addItem`.
+
+```typescript
+ItemsHolder.addItem("color", { value: "red" });
+```
```

Comment:
The pattern of alternating code snippets and explanations is taking up quite a lot of space. Try combining snippets.